src/dex/swaap-v2/swaap-v2.ts

```diff
@@ -335,13 +360,15 @@ export class SwaapV2 extends SimpleExchange implements IDex<SwaapV2Data> {
       );
 
       const prices = levelEntries.map((askAndBids: SwaapV2PriceLevels) => {
+
         const unitPrice = this.computePricesFromLevelsBids(
           [unitVolume],
           askAndBids,
           normalizedSrcToken,
           normalizedDestToken,
           side,
         )[0];
```

Comment:
I think here you might receive empty array => take first entry => get undefined and put it into `unit` place. But it is not impacting anything, as far as I can see